botocore/data/elbv2/2015-12-01/waiters-2.json

```diff
@@ -0,0 +1,100 @@
+{
+  "version": 2,
+  "waiters": {
+    "LoadBalancerExists": {
+      "delay": 15,
+      "operation": "DescribeLoadBalancers",
+      "maxAttempts": 40,
+      "acceptors": [
+        {
+          "matcher": "status",
+          "expected": 200,
+          "state": "success"
+        },
+        {
+          "matcher": "error",
+          "expected": "LoadBalancerNotFound",
+          "state": "retry"
+        }
+      ]
+    },
+    "LoadBalancerAvailable": {
+      "delay": 15,
+      "operation": "DescribeLoadBalancers",
+      "maxAttempts": 40,
+      "acceptors": [
+        {
+          "state": "success",
+          "matcher": "pathAll",
+          "argument": "LoadBalancers[].State.Code",
+          "expected": "active"
+        },
+        {
+          "state": "retry",
+          "matcher": "pathAny",
+          "argument": "LoadBalancers[].State.Code",
+          "expected": "provisioning"
+        },
+        {
+          "state": "retry",
+          "matcher": "error",
+          "expected": "LoadBalancerNotFound"
+        }
+      ]
+    },
+    "LoadBalancersDeleted": {
+      "delay": 15,
+      "operation": "DescribeLoadBalancers",
+      "maxAttempts": 40,
+      "acceptors": [
+        {
+          "state": "retry",
```

Comment:
Another unneeded acceptor. Would be nice to fix but should not block pulling this in.